src/main/resources/application-dev.yml

```diff
@@ -9,3 +9,6 @@ jobs:
   remove-vca-initial-retention-period-data:
     name: "RemoveVcaInitialRetentionPeriodData"
     cron: "0 */10 * ? * *" # every 10 minutes
+  remove-vca-final-retention-period-data:
+    name: "RemoveVcaFinalRetentionPeriodData"
+    cron: "0 */10 * ? * *" # every 10 minutes
```

Comment:
What about this cron expression "0 5/10 7-19 ? * MON-FRI" to try and avoid log output from other jobs by starting at 5 past the hour.
`At second :00, every 10 minutes starting at minute :05, every hour between 07am and 19pm, every day between Monday and Friday, every month `